torchgeo/datasets/oscd.py

```diff
@@ -362,9 +365,10 @@ def __init__(
         self,
         root_dir: str,
         bands: str = "all",
-        batch_size: int = 64,
+        batch_size: int = 32,
```

Comment:
This maybeee should be called `train_batch_size` with documentation in the constructor explaining that the test and val dataloader will use batch size of 1.